src/mumble/CoreAudio.mm

```diff
@@ -15,6 +19,380 @@
 // Ignore deprecation warnings for the whole file, for now.
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 
+
+namespace {
+extern "C" {
+// The dirty hack used to disable the "compulsory" ducking offered by VoiceProcessingAU.
+// A popular variant of this hack is `printf "p *(char*)(void(*)())AudioDeviceDuck=0xc3\nq" | lldb -n FaceTime`
+// See https://github.com/chromium/chromium/blob/6acb61fb1f335a720c51ed20acec5b3a4a19f308/media/audio/mac/audio_low_latency_input_mac.cc#L38
+// for reference.
+OSStatus AudioDeviceDuck(AudioDeviceID inDevice,
+		     Float32 inDuckedLevel,
+		     const AudioTimeStamp* __nullable inStartTime,
+		     Float32 inRampDuration) __attribute__((weak_import));
+}
+
+void UndoDucking(AudioDeviceID output_device_id) {
+    if (AudioDeviceDuck != nullptr) {
+	    // Ramp the volume back up over half a second.
+	    qDebug() << "CoreAudioInput: Undo Ducking caused by VoIP AU.";
```

Comment:
Iirc using the `<<` operator can cause errors when doing a Release build since by default the respective header is not included then :thinking: 
```suggestion
	    qDebug("CoreAudioInput: Undo Ducking caused by VoIP AU.");
```

Reply:
Wow, I'm surprised. Will do.